charts/capi/README.md

```diff
@@ -0,0 +1,30 @@
+# STFC Cloud CAPI Chart
+
+This chart deploys a Cluster-API (CAPI) K8s Cluster intended to be run on the STFC Cloud. 
+
+# Requirements 
+
+- An openstack project on the STFC Cloud with enough quota.
+- A dedicated floating IP on the project
+- An ubuntu VM 
+
+1. Setup an application credential and download clouds.yaml 
+
+2. SSH into your ubuntu VM
+
+3. Run Bootstrap script
+```bash
+# in repo root
+./scripts/bootstrap-capi-cluster.sh --floating-ip 130.x.x.x --app-cred-fp /path/to/clouds.yaml
```

Comment:
We should make it clear that `floating-ip` will be used for the control plane / kubeAPI not ingress, such as

> 3. Run bootstrap script. The `floating-ip` is used for <> and needs to be manually allocated (if one has not been).... . Documentation on obtaining an app cred can be found at <confluence docs>

To give people the best chance, the previous method relied on the files being complex enough (user-values.yaml) to force users to look at the docs but I think this is simple enough we need some steer at this level too